apps/lib/publisher_agent/config_test.go

```diff
@@ -21,3 +22,23 @@ func TestLoadConfig(t *testing.T) {
 	assert.Equal(t, "", config.PullBasedWsUrl)
 	assert.Equal(t, 5216, config.IncomingWsPort)
 }
+
+func TestLoadKeysFileFromEnv(t *testing.T) {
```

Comment:
Can you add another test to ensure the cascading prioritization works as expected?